admin-manual/installation-setup/installation/install-centos.rst

```diff
@@ -0,0 +1,340 @@
+
+     * Forensic tools repo
+
+       .. code:: bash
+
+          sudo rpm -Uvh https://forensics.cert.org/cert-forensics-tools-release-el7.rpm
+
+   * Then install the package:
+
+     .. code:: bash
+
+        sudo -u root yum install -y archivematica-mcp-client
+
+   * The MCP Client expects some programs in certain paths, so we put them in place:
+
+     .. code:: bash
+
+        sudo ln -s /usr/bin/7za /usr/bin/7z
+
+   * Tweak ClamAV configuration:
+
+     .. code:: bash
+
+        sudo -u root sed -i 's/^#TCPSocket/TCPSocket/g' /etc/clamd.d/scan.conf
+        sudo -u root sed -i 's/^Example//g' /etc/clamd.d/scan.conf
+
+   * Indexless mode:
+
+     If you are planning on running Archivematica in indexless mode (i.e.,
+     without Elasticsearch), then modify the relevant systemd EnvironmentFile
+     files by adding lines that set the relevant environment variables to
+     ``false``:
+
+     .. code:: bash
+
+         sudo sh -c 'echo "ARCHIVEMATICA_DASHBOARD_DASHBOARD_SEARCH_ENABLED=false" >> /etc/sysconfig/archivematica-dashboard'
+         sudo sh -c 'echo "ARCHIVEMATICA_MCPSERVER_MCPSERVER_SEARCH_ENABLED=false" >> /etc/sysconfig/archivematica-mcp-server'
+         sudo sh -c 'echo "ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_SEARCH_ENABLED=false" >> /etc/sysconfig/archivematica-mcp-client'
+
+   * After that, we can enable and start/restart services
+
+     .. code:: bash
+
+        sudo -u root systemctl enable archivematica-mcp-client
+        sudo -u root systemctl start archivematica-mcp-client
+        sudo -u root systemctl enable fits-nailgun
+        sudo -u root systemctl start fits-nailgun
+        sudo -u root systemctl enable clamd@scan
+        sudo -u root systemctl start clamd@scan
+        sudo -u root systemctl restart archivematica-dashboard
+        sudo -u root systemctl restart archivematica-mcp-server
+
+7. Finalizing installation
+
+   **Configuration**
+
+   Each service has a configuration file in
+   /etc/sysconfig/archivematica-packagename
+
+   **Troubleshooting**
+
+   If IPv6 is disabled, Nginx may refuse to start. If that is the case make sure
+   that the listen directives used under /etc/nginx are not using IPv6 addresses
+   like [::]:80.
+
+   CentOS will install firewalld which will be running default rules likely
+   blocking ports 81 and 8001. If you are not able to access the dashboard and
+   Storage Service, check if firewalld is running. If it is, you will likely
+   need to modify the firewall rules to allow access to ports 81 and 8001 from
+   your location:
+
+   .. code:: bash
+
+      sudo firewall-cmd --zone=public --add-port=81/tcp  --permanent
+      sudo firewall-cmd --zone=public --add-port=8001/tcp  --permanent
+      sudo service firewalld restart
+
+8. Complete :ref:`Post Install Configuration <post-install-config>`.
+
+.. _centos-post-install-config:
+
+Post Install Configuration
```

Comment:
Nit: we're not very consistent but I think we've been using less capitalization of headings. FWIW, we don't in AtoM. Also shouldn't this read `Post-install configuration` instead?

Reply:
Yeah, sentence-casing the titles is something I've been trying to do whenever I see something that is title-cased - thanks for catching this.